packages/react/src/primitives/Icon/deprecationWarning.ts

```diff
@@ -0,0 +1,10 @@
+export const deprecationWarning = (iconName) => {
```

Comment:
Should we make this more generic and add it to `utils`? IT seems like we will be doing this a fair amount as we go along. It would be nice to be able to search for all the `deprecationWarning` calls in the codebase as well.